src/rpc/rawtransaction.cpp

```diff
@@ -1323,8 +1324,8 @@ UniValue scanforunspent(const UniValue& params, bool fHelp)
                         if (!txindex.vSpent[dummy.n].IsNull())
                             continue;
 
-                        // Add to multimap
-                        uMultisig.insert(std::make_pair(txout.nValue, std::make_pair(tx.GetHash(), j)));
+                        // Add to deque
+                        Multisig.emplace_back(std::make_tuple(tx.GetHash(), j, txout.nValue, pblkindex->nHeight));
```

Comment:
Couldn't you have used a plain vector here instead of deque? The order is enforced by the while loop and the pblkindex->pnext, so it is in height order, equivalent to pblkindex->nHeight. In this way vector and deque are identical. There don't appear to be any emplace fronts here which would necessitate deque.

Reply:
yes ill change it this week, Originally the block number was not the first number then once the deque was made i refactored the order I stored the data.

Reply:
Think i also took into consideration that there can be multiple vouts in a block for some reason I thought that if pushing another matching key would over write the previous value stored. this on my list this week anyway

Reply:
Foggy treatment brain